### PR TITLE
Mercat recipe

### PR DESCRIPTION
**Pull request by ajaypanyala**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by daler** [comment]
@ajaypanyala please don't modify the build environment by using apt-get install. Any dependencies should be listed in the recipe. In this case, add `pyqt` to the requirements section of the recipe.

**Comment by ajaypanyala** [comment]
Thank you Ryan! I changed it back.

-Ajay

File: recipes/mercat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - python
+    - setuptools
+
+  run:
+    - python
+    - scikit-bio
+    - prodigal
+    - trimmomatic
+    - dask
+    - setuptools
```
Comment (bgruening) [inline recipes/mercat/meta.yaml]: Is setup tools really needed as a runtime dependency?

File: recipes/mercat/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    - pandas
+    - numpy
+    - humanize
+    - plotly
+    - psutil
+    - joblib
+
+
+#test:
```
Comment (bgruening) [inline recipes/mercat/meta.yaml]: Please include a test.

**Comment by druvus** [comment]
Please avoid usin git_url. A regular url with a checksum is recommended. You could do somethin like this for package, source and build sections:

```
{% set name = "mercat" %}
{% set version = "0.1" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

build:
  skip: True  # [not py27]
  number: 0
  entry_points:
    - mercat=mercat.mercat:mercat_main

source:
  fn: {{ name|lower }}_{{ version }}.tar.gz
  url: https://github.com/pnnl/mercat/archive/0.1.tar.gz
  md5: bce67a7bcb58c8daed4a00b377dad097
```

File: recipes/mercat/meta.yaml (hunk 1 above)
Comment (ajaypanyala) [inline recipes/mercat/meta.yaml]: It's not needed. Will remove that. Thanks!

File: recipes/mercat/meta.yaml (hunk 2 above)
Comment (ajaypanyala) [inline recipes/mercat/meta.yaml]: A simple test passes on MacOSX build, but not for Linux. When trying to import python modules, I get the error  
> ImportError: libstdc++.so.6: cannot open shared object file
I tried specifying libgcc, gcc as runtime dependencies in the recipe, but I still see it.
[https://travis-ci.org/ajaypanyala/bioconda-recipes/jobs/203529745](https://travis-ci.org/ajaypanyala/bioconda-recipes/jobs/203529745)

**Comment by druvus** [comment]
@ajaypanyala please take a look at the bioconda contributing guidelines and especially the [checklist](https://bioconda.github.io/guidelines.html#bioconda-recipe-checklist). It is probably a reason why a test is failing so skipping the test section and merging after @bgruening requested tests might not be the best strategy.

**Comment by ajaypanyala** [comment]
@druvus @bgruening  Sorry about the merge. I will fix the test section, run travis build again and merge it once they pass. Thanks!

**Pull request merged**